### PR TITLE
Improvements for README.md

### PR DESCRIPTION
**Pull request by vbrandl**
Some improvements for `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -176,7 +176,7 @@ That's it! Everything else is optional. Read on to learn how to configure specif
 
 ### Section Menu
 
-Theme uses [Section Menu for Lazy Bloggers](https://gohugo.io/extras/menus/#section-menu-for-the-lazy-blogger) to produce global site navigation, if enabled.
```
Comment (vbrandl) [inline README.md]: If the `if` clause is in the end of the sentence, there is no comma

File: README.md [hunk 2]
```diff
@@ -237,7 +237,7 @@ Additional information and examples, including how to set-up and use LQIP (Low-Q
 
 Promote more of your content to your site visitors. By offering your readers more content that's relevant to them you can increase your site's page views, the time spent on your site and reader loyalty.
 
-Related content surfaces content across sections by matching taxonomy tags. If After Dark finds related content it will automatically output a list of links to that content in reverse chronological order below the byline of your post content.
```
Comment (vbrandl) [inline README.md]: There is a comma if the `if` clause is at the beginning of the sentence ;)

File: README.md [hunk 3]
```diff
@@ -374,13 +374,13 @@ To customize default blocking configure the `noindex_kinds` setting in the `[par
   ]
 ```
 
-To block individual pages from being indexed add `nofollow` to your page's front matter and set the value to `true`, like:
+To block individual pages from being indexed add `noindex` to your page's front matter and set the value to `true`, like:
```
Comment (jhabdas) [inline README.md]: Nice catch!

File: README.md [hunk 4]
```diff
@@ -414,7 +414,7 @@ See the Hugo docs for additional [configuration options](http://gohugo.io/overvi
 
 Keep your content <abbr title="Don't Repeat Yourself">DRY</abbr> and improve thematic consistency across your site. To help achieve this, Hugo provides [Shortcodes](https://gohugo.io/extras/shortcodes).
 
-Shortcodes are very powerful, and can be used to achieve functionality not otherwise available in the markdown processor. Hugo provides a number of [built-in shortcodes](https://gohugo.io/extras/shortcodes#built-in-shortcodes) you can use on your site. And After Dark provides some as well.
```
Comment (vbrandl) [inline README.md]: Comma after and only in enumerations with three or more elements or when joining two independent sentences.

**Comment by jhabdas** [comment]
Closed by https://github.com/comfusion/after-dark/commit/b85fc9a53f3dc0a7a59dd6718da3f098c3050c61. Thank you very much @vbrandl.

**Pull request closed without merging**